src/physics_body.js

```diff
@@ -0,0 +1,35 @@
+import Vec2 from 'vec2';
+import Polygon from 'polygon';
+
+export default class PhysicsBody {
+    constructor(polygon, position, velocity, accel) {
+        this.position = position.slice(); //[x,y] origin bottom left
+        this._polygon = polygon.translate(polygon.center.negate(), true);
+        this.velocity = velocity.slice();//[units/s on x-axis, units/s on y-axis]
+        this.accel = accel;// m/(s^2)
```

Comment:
Non-pedantic: acceleration should be a vector, (I guess it should be named force then), but accel might not necessarily be gravity (e.g. wind).